fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -1238,6 +1238,13 @@ private void handleInsertStmt() throws Exception {
                     }
                 }
 
+                if (filteredRows > 0 &&
+                        (1 / (1 + (double) loadedRows / filteredRows)) > Config.default_max_filter_ratio) {
```

Comment:
I didn't even know that we have this config....
And I find that it only be used in the old load job, which is deprecated...

So I think better not use it. May be we can add a new session variable for insert operation?

Reply:
Done